autopromote/README.md

```diff
@@ -41,3 +41,9 @@ Format: `{"hour": int, "minute": int}`
 __enforce_force_install_time__: Have you decided 4:30 is a bad force install time? Set this value and all pkginfos, once parsed, will have their force_install_after_date changed to reflect the it.
 
 __force_install_denylist__: A list of pkginfos (as defined in their `name` attribute) on which no force_install_after_date will ever be set.
+
+__channels__: Channels allow one to specify a faster or slower promotion schedule for specific packages. This is a dictionary of channel names and an int or float multiplier:
+
+`{"channels": {"slow": 2.5}}` - this channel configuration would cause any packages in the "slow" channel to be promoted 2.5 times *slower*. This is achieved by multiplying the current promotion period by the channel's value. For faster promotion schedules, specify a float modifier of less than 1.
```

Comment:
Just for my own clarity: This means that a channel configuration like this...

```
{"channels": {"fast": 0.5}}
```
...would make any items associated to that channel to be promoted at a rate that is 2x _faster_ than the base-level promotion schedule?

If that is, indeed, the case, maybe you could add a quick little ditty like that ☝️ to this `README.md` to explicitly denote this type of configuration for hastening a promotion cadence?

Reply:
Correct - added a note clarifying this.